CMakeLists.txt

```diff
@@ -1062,6 +1062,11 @@ if(HAVE_FSETXATTR)
   endforeach()
 endif()
 
+check_symbol_exists(sa_family_t    "${CURL_INCLUDES}" HAVE_SA_FAMILY_T)
+if(HAVE_SA_FAMILY_T)
+  add_definitions(-DCURL_SA_FAMILY_T=sa_family_t)
```

Comment:
Why this approach? cmake typically sets all other defines so that `curl_config.h` gets made from the `curl_config.h.cmake` template.

Reply:
Sorry, this must have come from my lack of detailed knowledge of this project.
I'm aware of the `curl_config.h*` files and have checked them, but they did not appear to me to be the likely place to put the `CURL_SA_FAMILY_T` symbol in.
Plus I saw `CURL_SA_FAMILY_T` defined in `configure.ac`, therefore assumed the equivalent CMake approach shall have it covered in this file.
I'll take a further look at this.